docs/UserGuide.md

```diff
@@ -1414,6 +1414,89 @@ Command Options: This command does not take in any arguments and will not proces
 
 #### 4.4.1 HelpCommand
 
+Get help for UniCa$h.
+
+Command: `help <COMMAND_WORD>`
+
+Command Argument: `<COMMAND_WORD>` is the command to get help for. If no 
+argument is specified, a general help message is shown as well as a pop up 
+containing a link to our User Guide.
+
+> To get a list of `<COMMAND_WORD>`, do `help` with no arguments
+
+##### Successful Execution
+
+###### Example 1
+
+> **Case**: Get general help.
+>
+> **Input**: `help`
+>
+> **Output**:
+> ```
+> Welcome to UniCa$h!
+>
+>For more detailed help on a command: help <command_word>
+>
+> Available Commands:
+> add_transaction
+> delete_transaction
+> edit_transaction
+> list
+> find
+> get
+> get_total_expenditure
+> summary
+>
+> clear_transactions
+> reset_unicash
+>
+> help
+> exit
```

Comment:
This list is now longer right? Since the addition of budgets features

Reply:
It seems the help command is not updated

Reply:
Oh, do we need to update it first?